plugins/api-docs/dev/customizations.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Chip } from '@material-ui/core';
+import { EntityRow } from '../src/components/ApiExplorerTable/defaults';
+import { TableFilter } from '@backstage/core';
+
+const domainColumn = { title: 'Domain', field: 'entity.metadata.domain' };
+const capabilitiesColumn = {
+  title: 'Capabilities',
+  field: 'entity.metadata.capabilities',
+  cellStyle: {
+    padding: '0px 16px 0px 20px',
+  },
+  render: ({ entity }: EntityRow) => (
+    <>
+      {entity.metadata.capabilities &&
+        (entity.metadata.capabilities as string[]).map(t => (
+          <Chip
+            key={t}
+            label={t}
+            size="small"
+            variant="outlined"
+            style={{ marginBottom: '0px' }}
+          />
+        ))}
+    </>
+  ),
+};
+
+export const customColumns = [
+  'Name',
```

Comment:
Might me nice to make this a record that can be keyed off a well known column name string for the built in columns. That way if the default columns change TS will tell you.

Also, using the title of the column seems brittle. Maybe the idea of a keyed map of columns would help so the column definition can change independent of this mapping.